src/ProjectTemplates/Web.ProjectTemplates/GrpcService-CSharp.csproj.in

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Grpc.AspNetCore" Version="${GrpcAspNetCoreVersion}" />
+    <PackageReference Include="Google.Protobuf" Version="3.22.0-rc2" />
```

Comment:
Note, this is hard coded because:

1. I don't want to update other places that use Google.Protobuf, such as Microsoft.AspNetCore.Grpc.JsonTranscoding, to use it.
2. It will be removed in a month or two.